examples/persistent-counters/handler.go

```diff
@@ -3,18 +3,23 @@ package main
 import (
 	"errors"
 	"github.com/anujga/dstk/pkg/ss"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 	"time"
 )
 
-func handleResponseElement(elem interface{}, response *string, e *error) {
+func handleResponseElement(elem interface{}, response *interface{}, e *error) {
```

Comment:
its better to stick with `interface{}, err` which is idiomatic. Even better would be to create a new struct with result, error and can potientally be augmented with new info in a backward compatible manner